src/stylesheets/components/_search.scss

```diff
@@ -86,6 +58,7 @@
     background-repeat: no-repeat;
       @media (min-width: 30em) {
         background-size: 1.5rem;
+        background-size: 24px;
```

Comment:
Do we need both background-size here?

Reply:
nope I just forgot to remove that, thanks for catching it!